driver-pulsar/pom.xml

```diff
@@ -27,7 +27,7 @@
 
         <dependency>
             <groupId>org.apache.pulsar</groupId>
-            <artifactId>pulsar-client</artifactId>
+            <artifactId>pulsar-client-original</artifactId>
```

Comment:
What does this change do?

Reply:
"pulsar-client" is a JAR that contains all of the dependencies of Pulsar, shaded, it is like an uber jar.

Reply:
without this change IntelliJ messes up.

I didn't check how we package drivers. is it a problem if this jar adds its own third party deps to the NB build ?

Reply:
@jshook I can revert these changes to pom.xml

Reply:
The artifacts need to be resolvable in a public repository or it will break the build for everybody.

Reply:
I see that this is also a public artifact. No issues on this from me.